include/openthread/thread_ftd.h

```diff
@@ -739,6 +739,19 @@ void otThreadRegisterNeighborTableCallback(otInstance *aInstance, otNeighborTabl
  */
 void otThreadSetCcmEnabled(otInstance *aInstance, bool aEnabled);
 
+/**
+ * This method sets flag informing that thread protocol version check VR should be skipped
+ * when deciding if device should operate as router.
+ *
+ * @note This API requires `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE`, and is only used by Thread Test Harness
+ *       to indicate that thread protocol version check VR should be skipped.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aSkip  TRUE if the VR check should be skipped, FALSE otherwise.
+ *
+ */
+void otThreadSetSkipThreadVersionCheck(otInstance *aInstance, bool aSkip);
```

Comment:
nit: how about reversing the polarity and use "enabled" rather than "skip"?
```suggestion
void otThreadSetThreadVersionCheckEnabled(otInstance *aInstance, bool aEnabled);
```

Reply:
That works for me. But remember to initialize `mThreadVersionCheckEnabled = true` in such case.

Reply:
And the command should be `tvcheck` without `skip`.

Reply:
@jwhui @simonlingoogle  Are You sure about this change?  `otThreadSetThreadVersionCheckEnabled` would mean that this check is not done by default. Also imagine invoking `tvcheck enable` to turn off the thread version check (a bit misleading).

Reply:
Yes, with the proposed name change, we would also set the default value to enabled.

Reply:
@jwhui @simonlingoogle should be fixed now.